[BUG] PNPM Unmet peer dependencies

When I try to install all dependencies using `pnpm` the installation is not completed successful due dependencies error:

```
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

components/github
└─┬ @octokit/plugin-paginate-rest
  └── ✕ unmet peer @octokit/core@>=4: found 3.6.0
```

With this error, it's not possible `push` commits because `husky` will fail.

--- a/components/github/package.json
+++ b/components/github/package.json
@@ -11,7 +11,7 @@
     "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
     "license": "MIT",
     "dependencies": {
-        "@octokit/core": "^3.6.0",
+        "@octokit/core": "^4.0.4",
         "@octokit/plugin-paginate-rest": "^2.17.0",
         "@octokit/webhooks-definitions": "^3.29.0",
         "@pipedream/platform": "^1.0.0"

